src/client/utils/getInfoFromHash.ts

```diff
@@ -1,8 +1,7 @@
-import isNaN from 'lodash/isNaN';
 import { hasInHash, getHashAsArray } from './handleHash';
 
-function filterNumbers(item: string): boolean {
-	return isNaN(parseInt(item, 10)) && item !== '';
+function consistsOnlyNumbers(item: string): boolean {
```

Comment:
Something like this would be more correct and clear:

```suggestion
function hasDigitsOnly(item: string): boolean {
```

Reply:
Thank you. Since I am not a native speaker so I am always welcoming this kind of suggestion.